infra/gcp/roles/specs/access-logs.viewer.yaml

```diff
@@ -0,0 +1,14 @@
+title: Access Logs
+description: View access to GCP access logs
+name: access-logs.viewer
+include:
+  roles:
+  - roles/compute.viewer
+  - roles/dns.reader
+  - roles/iam.securityReviewer
+  - roles/resourcemanager.organizationViewer
+  - roles/serviceusage.serviceUsageConsumer
```

Comment:
Why are these roles necessary?

Reply:
I'm not particularly sure which roles may be needed, but would like to start the conversation